### DISCUSSION TITLE
What is our name?

### DISCUSSION THREAD

**Discussion by dmjacobsen** [bikeshack/community#9] [General]
As was discussed in the board meeting this morning, the provisional name "bikeshack" was discussed, and I think we came around to calling this collaboration "OpenCSM" where the "C" stands for something other than "Cray", perhaps "Compute".

Anyway, discuss here!

**Comment by alexlovelltroy** [comment]
Here are the options that have been proposed so far:

- Open Community System Manager
- Open Cloud native System Manager
- Open Central System Manager
- Open Cloud Supercomputer Manager

**Comment by trandles-lanl** [comment]
I get wanting to keep it "CSM" and redefine the acronym but I think in practice "CSM" will be "CSM" to people regardless of what the C means. This will be heavily biased by their current understanding (or misunderstanding) of CSM and will be an uphill marketing battle.

**Reply by dmjacobsen** [reply]
I had kind of thought that we would want to abandon the name "CSM" and had at least a placeholder on some internal NERSC code that i'll upload when it becomes relevant/possible calling it "CMS" again (like the original internal name i've heard Cray people use).

Compute Management System
Comical Math Solver

I think we can also just pick some other single word we like that doesn't have negative word associations:

Avacado

**Comment by alexlovelltroy** [comment]
I'll let @larry-kaplan weigh in as well, but my understanding is that HPE prefers retaining the CSM acronym and plans to continue existing support under that name.

We did have some other names added to our discussion without the CSM baggage:

- **Modular HPC System Manager** (MHSM): This title emphasizes the modularity of the system, which was an original design goal, but we can take it much further.
- **Confidential Application and Resource Management (CARMA)**: This title emphasizes the future focus on confidential computing and the ongoing task of resource management, with a nod to "karma" - creating good outcomes through secure and efficient system management.
- **ExaMachina**: Combining 'Exa', referring to the Exascale capability, and 'Machina', the Latin word for machine, this name highlights the software's high-performance capability and its suitability for large, powerful machines.
- **Hyperion**: Named after one of the Greek Titan gods, Hyperion, the father of sun, moon, and dawn, suggests power, illumination, and the start of a new day or era. It implies the capability of your software to bring about new beginnings in the realm of HPC.

**Reply by larry-kaplan** [reply]
If you are saying that HPE will continue to support the CSM product, that's true.  For this project, HPE doesn't have a preference that I'm aware of, other than not use "Cray" as part of the definition.

I'm a bit more of a fan of the acronym style of name though single work names can be OK, they just seem harder to make relevant.  So far, OCSM and MHSM are the leaders for me.
